docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md

```diff
@@ -26,11 +26,13 @@ If the amount of data on your disk is much less than the size of the disk, then
 
 ## Download a Disk over SSH
 
-### Boot into Rescue Mode
+This guide was written to support Downloading a disk over SSH from a  **Linode**, which will be used to Create and Send an image to a separate receiving system that you personally own. This means that the guide will have you switching between the execution of commands on both the **Origin Linode** as the origin of your disk image, and the **Receiving System** which will receive a copy of this image. You will see the terms Origin Linode and Receiving System used to describe these two separate environments.
```

Comment:
* Please remove usage of the future tense
* Downloading, Receiving System, and Origin Linode do not need to be capitalized.
* Rewrite suggestion for the first sentence: This guide shows you how to download a Linode's disk image over SSH to a separate receiving system, like a personal computer.